litellm_proxy/README.md

```diff
@@ -0,0 +1,20 @@
+# LiteLLM Proxy Docker
+
+All of our embeddings and LLM calls will now pass through this proxy. See [confluence page](https://idinsight.atlassian.net/wiki/spaces/PD/pages/2452488357/LiteLLM+Proxy+Server) for details.
```

Comment:
This is a private document :o

Reply:
Yeah it's temporary for the draft PR - will remove.